jupyterlab_gcedetails/src/components/chart_wrapper.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { AreaChart, Area, Tooltip, CartesianGrid, YAxis } from 'recharts';
+
+interface ChartProps {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any[];
+  dataKey: string;
+  title: string;
+  titleClass: string;
+  areaChartProps: {};
+  areaProps: {};
+  cartesianGridProps: {};
+  yAxisProps: {};
```

Comment:
should we have more specific types defined here for our current use or are you trying to make the component more general?

Reply:
trying to make it more general, but I will update to give the objects more specific shape in this PR since the props are pretty well defined from the recharts api

Reply:
my thinking is that eventually if this becomes a shared component we would have different base configurations of the props set so everything is stylistically similar, and people can override them as needed.